yastd/share/dbus.conf

```diff
@@ -11,6 +11,8 @@
     <allow send_destination="org.opensuse.YaST" />
     <allow send_destination="org.opensuse.YaST"
            send_interface="org.freedesktop.DBus.Introspectable"/>
+    <allow send_destination="org.opensuse.YaST.Installer1" />
+    <allow receive_sender="org.opensuse.YaST.Installer1" />
```

Comment:
Still needed?

Reply:
well, it is needed as it follows correct versioned interface. I just do not want to kill old one if you switch between branches.

Reply:
Yes, you are right, I did not know about the versioning. I wonder if we should drop the lines below:

```xml
    <allow send_destination="org.opensuse.YaST.Installer" />
    <allow receive_sender="org.opensuse.YaST.Installer" />
```

Reply:
We will need to review the D-Bus policy in the future anyway, so let's keep it as it is by now. Thanks!